subsys/net/ip/net_context.c

```diff
@@ -2166,6 +2180,113 @@ static int set_context_proxy(struct net_context *context,
 #endif
 }
 
+static int set_context_nodelay(struct net_context *context,
```

Comment:
Why is a dummy NODELAY implementation part of this PR? Zephyr does not implment it so it should rather return `ENOPROTOOPT` as it used to IMO.